test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/BodyModelBinderTests.cs

```diff
@@ -820,7 +820,8 @@ private class TestableXmlSerializerInputFormatter : XmlSerializerInputFormatter
         {
             private bool _throwNonInputFormatterException;
 
-            public TestableXmlSerializerInputFormatter(bool throwNonInputFormatterException)
+            public TestableXmlSerializerInputFormatter(bool throwNonInputFormatterException) :
+                base(new MvcOptions())
```

Comment:
Please put the colon on the same line as `base` (or `this`) invocations (throughout PR).